lib/services/playon_handler.dart

```diff
@@ -122,8 +122,13 @@ class PlayonHandler {
         Stream.periodic(const Duration(seconds: 5), (count) {
       return count;
     }).listen((count) {
-      startListener();
-      _playOnHandlerLogger.info("Attempted to restart listener");
+      if (count < 24) { // We try to connect for two minutes before giving up
+        _playOnHandlerLogger.warning("Attempt $count to restart playon listener");
+        startListener();
+      } else {
+        _playOnHandlerLogger.warning("Stopped attempting to connect playon");
+        _reconnectionSubscription?.cancel();
+      }
```

Comment:
This is fine for now, but once #1139 lands we can probably try to reconnect once the connection status changes.  
It would be a bit weird if we never tried again until a restart.  
Alternatively, after 24 counts you could increase the duration we wait, from 5 seconds to 30 or even 60 seconds, instead of completely giving up.